### DISCUSSION TITLE
Mehr Geld

### DISCUSSION THREAD

**Discussion by BigDBudda** [Royal-Modding/FS19_BetterContracts#33] [Q&A]
Hi, nur mal als Frage, besteht die Möglichkeit den Geld Multiplier zu erhöhen damit man etwas mehr bekommt für die missionen? Ich habe teilweise Missionen die nur knapp 4000€ geben.

**Comment by sashaman** [comment]
with seasons and precision farming i sometimes get negative values on reward, dunno if this is intended. 
also, whatever i do, i cannot get my own value modifiers to work, seems script calculates its own rewards? 
f.ex. 
```lua
FertilizeMission.REIMBURSEMENT_PER_HA = 750.000000
FertilizeMission.REWARD_PER_HA = 250.000000
BaleMission.REWARD_PER_HA_HAY = 10000.000000
BaleMission.REWARD_PER_HA_SILAGE = 8500.000000
CultivateMission.REWARD_PER_HA = 2400.000000
HarvestMission.REWARD_PER_HA_SMALL = 4800.000000
HarvestMission.REWARD_PER_HA_WIDE = 1800.000000
PlowMission.REWARD_PER_HA = 3000.000000
SowMission.REWARD_PER_HA = 2600.000000
SprayMission.REWARD_PER_HA = 250.000000
SprayMission.REIMBURSEMENT_PER_HA = 750.000000
```
